### PR TITLE
Deploy token

### PR DESCRIPTION
**Pull request by Kouprin**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,12 +30,12 @@
     "node-fetch": "^2.6.0",
     "pm2": "^4.4.0",
     "pm2-promise": "^2.0.1",
-    "rainbow-bridge-lib": "near/rainbow-bridge-lib#8e5567c0277d13430f952a19bb5e16ceb0b4392c",
+    "rainbow-bridge-lib": "near/rainbow-bridge-lib#373e9f787e51703f186e83e8036dc35131742c1b",
```
Comment (ailisp) [inline package.json:33]: I see, this is a good way to test rainbow-bridge-lib on ci. There's another way i used:
set `PATCH_RAINBOW_BRIDGE_LIB=373e9f787e51703f186e83e8036dc35131742c1b` env var and run a manual build on rainbow-bridge-cli pipeline on buildkite

File: package.json [hunk 2]
```diff
@@ -30,12 +30,12 @@
...
     "rainbow-bridge-rs": "1.0.2",
     "rainbow-bridge-sol": "1.1.0",
     "request": "^2.88.2",
     "web3": "=1.2.6",
-    "rainbow-token-connector": "near/rainbow-token-connector#620ad3e45f9f722d77f88a7a39b252d5a4ead9ef"
+    "rainbow-token-connector": "near/rainbow-token-connector#0f874679dc534bfe9cbe8092584a5594a299fda1"
```
Comment (ailisp) [inline package.json:38]: Also add this to ci/patch_contract_version.js? (btw the name seems more proper to be patch_rainbow_deps.js or similar)

File: test/run_test.sh [hunk 3]
```diff
@@ -10,7 +10,7 @@ ROOT_DIR=$TEST_DIR/..
 trap cleanup EXIT
 cleanup() {
     cd $ROOT_DIR
-    node index.js stop all
+    #node index.js stop all
```
Comment (ailisp) [inline test/run_test.sh:13]: is this file still used anywhere? Looks we can delete it

File: yarn.lock [hunk 4]
```diff
@@ -4068,12 +4048,9 @@ rainbow-bridge-sol@1.1.0:
...
 
-rainbow-token-connector@near/rainbow-token-connector#620ad3e45f9f722d77f88a7a39b252d5a4ead9ef:
-  version "0.1.0"
-  resolved "https://codeload.github.com/near/rainbow-token-connector/tar.gz/620ad3e45f9f722d77f88a7a39b252d5a4ead9ef"
-  dependencies:
-    rainbow-bridge-lib "https://github.com/near/rainbow-bridge-lib"
-    tweetnacl "^1.0.3"
+rainbow-token-connector@near/rainbow-token-connector#0f874679dc534bfe9cbe8092584a5594a299fda1:
+  version "0.0.0"
+  resolved "https://codeload.github.com/near/rainbow-token-connector/tar.gz/0f874679dc534bfe9cbe8092584a5594a299fda1"
```
Comment (ailisp) [inline yarn.lock:4053]: This looks correctly resolved to same rainbow-bridge-lib, nice!

**Review by ailisp — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (Kouprin) [reply package.json]: Done

File: test/run_test.sh (hunk 3 above)
Comment (Kouprin) [reply test/run_test.sh:13]: It's for local testing. Let's keep it for now.

**Pull request merged**